onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -718,7 +760,9 @@ void addGlobalMethods(py::module& m, const Environment& env) {
             onnxruntime::CreateExecutionProviderFactory_NGraph("CPU"),
 #endif
 #ifdef USE_OPENVINO
-            onnxruntime::CreateExecutionProviderFactory_OpenVINO(openvino_device),
+            onnxruntime::CreateExecutionProviderFactory_OpenVINO(openvino_device_type,
```

Comment:
this code path is not really useful for openvino ep, and ends up being a no-op.
just specify the default values as the arguments here instead of referencing the globals (so we can get rid of the globals)

Reply:
done